HACKING.md

```diff
@@ -0,0 +1,95 @@
+# Setting up the image
+
+We recommend using [the images](https://aiyprojects.withgoogle.com/voice) we
+provide. Those images are based on [Raspbian](https://www.raspberrypi.org/downloads/raspbian/),
+with a few customizations and are tested on the Raspberry Pi 3. If you prefer
+to setup Raspbian yourself, there are some manual steps you need to take.
+
+## Installing the dependencies
+
+First, make sure you have `git` installed and clone this repository in
+`~/voice-recognizer-raspi`:
+
+```shell
+sudo apt-get install git
+cd
+git clone https://github.com/google/aiyprojects-raspbian.git voice-recognizer-raspi
+```
+
+Then, install the project dependencies and setup the services:
+
+``` shell
+cd ~/voice-recognizer-raspi
+scripts/install-deps.sh
+sudo scripts/install-services.sh
+```
+
+## Installing the Voice HAT driver and config
+
+To use the Voice HAT, you'll need to upgrade your kernel to 4.9, then adjust the
+kernel and ALSA configuration:
+
+``` shell
+sudo apt-get update
+sudo apt-get install raspberrypi-kernel
+sudo scripts/configure-driver.sh
+sudo scripts/install-alsa-config.sh
+sudo reboot
+```
+
+## Get cloud credentials
+
+To access the cloud services you need to register a project and generate
+credentials for cloud APIs. This is documented in the
+[setup instructions](https://aiyprojects.withgoogle.com/voice#users-guide-1-1--connect-to-google-cloud-platform) on the
+webpage.
+
+# Making code changes
```

Comment:
This heading doesn't make sense - not all of the following text should be under "Making code changes". Maybe just add "## Running automatically" before the service bit.

Reply:
I wasn't sure whether there should be 1 or two hash signs before "Making code changes" to make it either a level 1 or a level 2 header. I changed it to two, because I didn't think "Running automatically" belonged under it.